README.md

```diff
@@ -53,6 +53,9 @@ de.remsfal.auth.oidc.client-secret=<YOUR-SECRET>
 de.remsfal.auth.session.secret=<YOUR-CUSTOM-SESSION-SECRET>
 ```
 
+## JWT Token
+For the JWT token its highly recommended to replace the default private key and public key with your own.
+
```

Comment:
Mentioning the replacement of the default JWT keys is good, but more details are needed. Please include instructions on how to generate your own keys, how to set them up in the configuration, and tips for keeping them secure. This will make the guidance clearer and more complete.